kimchi/src/circuits/expr.rs

```diff
@@ -2168,12 +2174,15 @@ pub mod constraints {
         Self: std::marker::Sized,
     {
         /// Double the value
+
```

Comment:
why the spaces there :o ?

Reply:
Done https://github.com/o1-labs/proof-systems/pull/752/commits/e74530b7cf67a9cf66787be09ff530dff090630e